Docs/source/install/cmake.rst

```diff
@@ -315,21 +315,16 @@ So another sophisticated example might be: use Clang as the compiler, build with
 Here we wrote this all in one line, but one can also set all environment variables in a development environment and keep the pip call nice and short as in the beginning.
```

Comment:
@roelof-groenewald I might want to simplify this further in a follow-up, as in https://impactx.readthedocs.io/en/latest/install/cmake.html